osu.Game.Rulesets.Taiko/UI/DrumTouchInputArea.cs

```diff
@@ -1,21 +1,26 @@
 // Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
+#pragma warning disable IDE0001 // Simplify Names
+
 using System.Collections.Generic;
 using System.Diagnostics;
 using osu.Framework.Allocation;
+using osu.Framework.Bindables;
 using osu.Framework.Extensions.Color4Extensions;
 using osu.Framework.Graphics;
 using osu.Framework.Graphics.Containers;
 using osu.Framework.Graphics.Shapes;
 using osu.Framework.Input.Bindings;
 using osu.Framework.Input.Events;
 using osu.Game.Graphics;
+using osu.Game.Rulesets.Taiko.Configuration;
 using osuTK;
 using osuTK.Graphics;
 
 namespace osu.Game.Rulesets.Taiko.UI
 {
+    using TaikoInput = TaikoAction; // Functionally identical to TaikoAction, it's just a readability thing
```

Comment:
This just makes the file more confusing to me, especially since they are synonymous with eachother.

Reply:
It exists purely to make the file more understandable
Having a visual split between what was an input by the user and what was interpreted by the game through the control scheme makes things like this much less weird
![image](https://user-images.githubusercontent.com/48618519/211640637-9e6125b4-1630-4221-a700-158b76745d1b.png)

Reply:
Please just don't. Literally no other file in the repository does this sort of inner-scoped using.

Reply:
Alright